ui/packages/app/web/src/components/ui/Navbar.tsx

```diff
@@ -164,7 +157,15 @@ const Navbar = () => {
                                   'rounded-none hover:no-underline border-b-2 focus:ring-0 focus:outline-none focus:ring-offset-0 h-full whitespace-nowrap font-medium'
                                 )}
                                 variant="link"
-                                onClick={() => compareProfile(navigateTo)}
+                                onClick={() =>
+                                  navigateTo(
```

Comment:
Is it not necessary to include the existing query params as well? 
So that the current state is retained if available, or does it work that already?

Reply:
it totally is necessary. I've added that bit now